monitoring/values.yaml

```diff
@@ -0,0 +1,246 @@
+global:
+  ingress:
+    # optional annotation overrides
+    annotations: {}
+
+grafanaIngress:
+  annotations:
+    alb.ingress.kubernetes.io/healthcheck-path: /api/health
+
+prometheusIngress:
+  annotations:
+    alb.ingress.kubernetes.io/healthcheck-path: /-/healthy
+
+grafana:
+  adminUser: admin
+  # An adminPassword must be set
+  # adminPassword: {password}
+
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+        - name: Prometheus
+          type: prometheus
+          url: http://urban-os-prometheus-server
+          access: proxy
+          isDefault: true
+        - name: CloudWatch
```

Comment:
Ditto to this inadvertently pinning us to AWS, once again not sure if there is a solution

Reply:
Moved CloudWatch datasource to urban-os-deploy in an effort not to pin us to aws in the public chart